bundles/org.eclipse.passage.lbc.equinox/src/org/eclipse/passage/lbc/internal/equinox/i18n/EquinoxMessages.properties

```diff
@@ -23,3 +23,4 @@ ServerConditionArbiter_i_not_reserved=Condition was not reserved
 ServerConditionArbiter_i_reserv=Condition in reserve
 ServerRunnerImpl_i_launcher_bind=Bind BackendLauncher {0}
 ServerRunnerImpl_i_launcher_unbind=Unbind BackendLauncher {0}
+ServerRunnerImpl_error_launching=[ERROR] Could not start BackendLauncher instance:"{0}".
```

Comment:
the prefix `[ERROR]` should be removed from the message itself as typically it is added by the logger